src/components/fabSpeedDial/fabSpeedDial.js

```diff
@@ -0,0 +1,130 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('material.components.fabSpeedDial', [])
+    .directive('mdFabSpeedDial', MdFabSpeedDialDirective);
```

Comment:
The convention *should* be that only _constructors_ are PascalCase, where a constructor is is something that's called with the `new` keyword. 

A naming convention (PascalCase here) is the only way to communicate in JS that something is meant to be used as a constructor intended for use with the `new` keyword. This is the way of the Google js style guide and, as far as I know, is generally the accepted pattern among most of the wider JS community.